lib/transaction.js

```diff
@@ -15,6 +15,8 @@ const Utils = require('./utils');
  * @param {String} options.type=true Sets the type of the transaction.
  * @param {String} options.isolationLevel=true Sets the isolation level of the transaction.
  * @param {String} options.deferrable Sets the constraints to be deferred or immediately checked.
+ * @param {String} options.readOnly=true Sets the read-only property of the transaction, allowing
```

Comment:
`options.readOnly=false Sets read-only property of transaction, such transaction will use read replicas when available`